File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.lua
-- comment.line.double-dash
--[[
comment.block
]]
PI = 3.14 -- constant.numeric
ESC = "\n" -- constant.character.escape
FLAG = true -- constant.language

-- @param c string
function myFunction(param) -- entity.name.function, variable.parameter
	local color = "#ff0000" -- constant.other, string.quoted.double
	if not FLAG then -- keyword.control
		return color -- keyword.control
	end
end

MyClass = {} -- entity.name.type

File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.py
# comment.line.number-sign
"""comment.block"""

PI = 3.14  # constant.numeric
ESC = "\n"  # constant.character.escape
FLAG = True  # constant.language


class MyClass(BaseClass):  # entity.name.type, entity.inherited-class
    def func(self, param):  # entity.name.function, variable.parameter
        color = "#ff0000"  # constant.other, string.quoted.double
        if FLAG:  # keyword.control
            return color  # keyword.control


__underline__  # markup.underline
print("bold text")  # markup.bold
